samples/DispatcherSample.Web/DispatcherSample.Web.csproj

```diff
@@ -1,17 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
It seemed like the simplest thing to do to update the sample to work with multiple startup classes instead of having separate projects.

Reply:
👍 

I think I did this before I understood how a single site could be initialized with different startup classes.